packages/mds-ingest-service/service/provider.ts

```diff
@@ -189,6 +189,10 @@ export const IngestServiceProvider: ServiceProvider<IngestService & IngestMigrat
     }
 
     try {
+      await IngestRepository.writeMigratedTelemetry(
```

Comment:
I think this won’t write the telemetry to cache/stream. I think you basically want to call writeMigratedTelemetry.